tools/Generator/Generator.cs

```diff
@@ -24,6 +24,7 @@ public void Run ()
 			LoadSpecification ();
 
 			GenerateEnums ();
+            GenerateStructs ();
```

Comment:
Indentation? It looks like you're using spaces instead of tabs. It causes the patch to look "weird" in context.